db/migrations/2008040105_rework_root_group.sql

```diff
@@ -1,4 +1,15 @@
 -- +migrate Up
+SELECT `id` INTO @root_id FROM `groups` WHERE `type`='Base' AND `text_id`='Root';
+SELECT `id` INTO @root_self_id FROM `groups` WHERE `type`='Base' AND `text_id`='RootSelf';
```

Comment:
What will happens if there are several ones?
As there may be several domains, we could have several roots, one per domain.

However, if there is only one in the anonymized we could consider it is ok this way (even if I would prefer it to fail if there are more than one instead of doing something completely wrong)

Reply:
There is only one group with (`type`='Base' AND `text_id`='Root') and only one group with (`type`='Base' AND `text_id`='RootSelf') in the anonymized DB. These SELECT statements would fail otherwise:
```
mysql> SELECT `id` INTO @root_id FROM `groups` WHERE `type`='Base';
ERROR 1172 (42000): Result consisted of more than one row
```